haproxy/values.yaml

```diff
@@ -0,0 +1,28 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/haproxytech/haproxy-debian
+  tag: 2.1
+  pullPolicy: IfNotPresent
+
+ingress:
+  enabled: false
+
+service:
+  port: 8080
```

Comment:
Since the default is an LDAP TLS proxy, I would change the port to 30636 as well.